src/content/protocols/aave.md

```diff
@@ -0,0 +1,885 @@
+| VotingPortal_Eth_Eth                                        | [0x6ACe1Bf22D57a33863161bFDC851316Fb0442690](https://etherscan.io/address/0x6ACe1Bf22D57a33863161bFDC851316Fb0442690) |
+| VotingPortal_Eth_Avax                                       | [0x9Ded9406f088C10621BE628EEFf40c1DF396c172](https://etherscan.io/address/0x9Ded9406f088C10621BE628EEFf40c1DF396c172) |
+| VotingPortal_Eth_Pol                                        | [0xFe4683C18aaad791B6AFDF0a8e1Ed5C6e2c9ecD6](https://etherscan.io/address/0xFe4683C18aaad791B6AFDF0a8e1Ed5C6e2c9ecD6) |
+| EmergencyRegistry                                           | [0x73C6Fb358dDA8e84D50e98A98F7c0dF32e15C7e9](https://etherscan.io/address/0x73C6Fb358dDA8e84D50e98A98F7c0dF32e15C7e9) |
+| CCIPAdapter                                                 | [0xB7a6618df58626C3a122ABAFD6Ee63Af63f3Ef29](https://etherscan.io/address/0xB7a6618df58626C3a122ABAFD6Ee63Af63f3Ef29) |
+| PolygonAdapterEthereum (native adapter)                     | [0x1562F1b2487F892BBA8Ef325aF054Fd157510a71](https://etherscan.io/address/0x1562F1b2487F892BBA8Ef325aF054Fd157510a71) |
+| LayerZeroAdapter                                            | [0x8410d9BD353b420ebA8C48ff1B0518426C280FCC](https://etherscan.io/address/0x8410d9BD353b420ebA8C48ff1B0518426C280FCC) |
+| HyperLaneAdapter                                            | [0x01dcb90Cf13b82Cde4A0BAcC655585a83Af3cCC1](https://etherscan.io/address/0x01dcb90Cf13b82Cde4A0BAcC655585a83Af3cCC1) |
+| SameChainAdapter                                            | [0x6cfbd2aA4691fc18B9C209bDd43DC3943C228FCf](https://etherscan.io/address/0x6cfbd2aA4691fc18B9C209bDd43DC3943C228FCf) |
+| CrossChainController (Proxy)                                | [0xEd42a7D8559a463722Ca4beD50E0Cc05a386b0e1](https://etherscan.io/address/0xEd42a7D8559a463722Ca4beD50E0Cc05a386b0e1) |
+| CrossChainControllerUpgradeRev3 (Implementation)            | [0x92f4736b72d131d836b3e4d4c3c23fe53150ce4d](https://etherscan.io/address/0x92f4736b72d131d836b3e4d4c3c23fe53150ce4d) |
+| PayloadsControllerDataHelper                                | [0xE3B770Dc4ae3f8bECaB3Ed12dE692c741603e16A](https://etherscan.io/address/0xE3B770Dc4ae3f8bECaB3Ed12dE692c741603e16A) |
+| GovernanceDataHelper                                        | [0x971c82c8316aD611904F95616c21ce90837f1856](https://etherscan.io/address/0x971c82c8316aD611904F95616c21ce90837f1856) |
+| VotingMachineDataHelper                                     | [0x77976B51569896523EE215962Ee91ff236Fa50E8](https://etherscan.io/address/0x77976B51569896523EE215962Ee91ff236Fa50E8) |
+| MetaDelegateHelper                                          | [0x94363B11b37BC3ffe43AB09cff5A010352FE85dC](https://etherscan.io/address/0x94363B11b37BC3ffe43AB09cff5A010352FE85dC) |
+
+## Permission owners
+
+| Name                                     | Account                                                                                                               | Type         |
```

Comment:
is the Aave Chan Initiative multisig missing here?

Reply:
let's discuss whether External rewards are in/out scope

Reply:
the rewards system is governed by the Aave DAO, e.g. `EmissionManager` and `RewardsControllerManager` are "owned" by the DAO, so this should be considered in-scope for now